content/php/concepts/math-functions/terms/decbin/decbin.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'decbin()'
+Description: 'Converts a decimal number to a binary number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+  - 'Math Functions'
```

Comment:
remove these 2. We list a max of 2 under subjects

Reply:
noted on this

will edit accordingly

thanks